docs/glossary.md

```diff
@@ -0,0 +1,21 @@
+| canvas             | The canvas in the center of napari viewer contains the visual display of the data passed to napari, including images, points, shapes, and other supported data types.                                                                                                                                                                                                                                                                                                                                                                                                                        |                                                                             |
+| channel            | A channel is an image where each pixel only contains its intensity value (i.e. grayscale image). In fluorescence microscopy, each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue (RGB). In napari, an image dimension that is not spatial or temporal can be considered as channels, and is typically represented as individual [image layer](https://napari.org/tutorials/fundamentals/image.html). See also [this wiki page](https://en.wikipedia.org/wiki/Channel_(digital_image)) for specific examples.       | raster band in geoscience                                                   |
+| colormap           | A colormap represents the value of each pixel as a particular color. In fluorescence microscopy, the colormap can be selected to mirror the color of the collected wavelength. In other applications, colormaps can be used to provide the visual representation of the measured signal. There are many different types of colormaps available, see [choosing colormaps in matplotlib](https://matplotlib.org/stable/tutorials/colors/colormaps.html).                                                                                                                                                                                                                                                                                                                                                                    | look up table, LUT                                                          |
+| hook               | Hook is the place within napari where we promise to call functions created by external developers and installed by the user. Plugin developers can use hooks to extend functionality of napari. See [this wiki page on hooking](https://en.wikipedia.org/wiki/Hooking) for additional details.                                                                                                                                                                                                                                                                                               |                                                                             |
```

Comment:
```suggestion
| hook               | Hooks are a way to tell napari "If this event happens, then call this function". [Hooking up your own events](https://napari.org/docs/dev/guides/connecting_events.html?highlight=hook). Plugin developers can also use hooks to extend functionality of napari. See the [napari hook specification reference](https://napari.org/docs/dev/plugins/hook_specifications.html) in the [napari plugin documentation](https://napari.org/docs/dev/plugins/index.html).                                                                                                                                                                                                                                                                                               |                                                                             |
```

Reply:
Reasons for suggestion:
1. Hooks are about *when* something happens, not *where*. I think it's much less confusing to describe it in those terms.
2. I think we should link to the napari docs pages on hooks & hookspecs, instead of Wikipedia. That's because for this concept, the reader will want to know how it applies specifically to napari (as opposed to the more general image analysis terms, where it does make sense to link to Wikipedia for a more general understanding, here it's unhelpful if the reader has only a general understanding but doesn't know how to apply that in napari).